ecommerce_integrations/unicommerce/order.py

```diff
@@ -78,8 +87,9 @@ def _get_new_orders(
 	for order in uni_orders:
 		if order["channel"] not in configured_channels:
 			continue
-		if frappe.db.exists("Sales Order", {ORDER_CODE_FIELD: order["code"]}):
-			continue
+		if check:
```

Comment:
Why is this added if it's never set to `False` anywhere?

Reply:
Removed